common-pg/src/main/scala/com/socrata/pg/store/RollupManager.scala

```diff
@@ -61,7 +62,7 @@ class RollupManager(pgu: PGSecondaryUniverse[SoQLType, SoQLValue], copyInfo: Cop
    * Once we have the analyzed rollup query with column names, we need to remove the leading "_" on non-system
    * columns to make the names match up with the underlying dataset schema.
    */
-  private def columnNameRemovePrefixMap(cn: ColumnName): ColumnName = {
+  private def columnNameRemovePrefixMap(cn: ColumnName, todo: Qualifier): ColumnName = {
     cn.name(0) match {
```

Comment:
can you pick a different variable name than `todo`, or at least add a comment of why it is called `"todo"`